tasks/main.yml

```diff
@@ -1,4 +1,17 @@
 ---
+- include: compat_ansible_admin_group.yml
+  when: ansible_admin_group is defined
+
+- name: Verify that the groups are all defined
+  fail:
+    msg: "ansible_admins_group is defined while ansible_committers_group is not"
+  when: ansible_admins_group is defined and not ansible_committers_group is defined
```

Comment:
"not is defined" is really hurting my head, so I would suggest using "is undefined"